src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Hi @KulaniT , could you give a bit more elaboration on what kind of comments I should be adding here? Sorry, might have missed this part of the lecture out. Thanks in advance.